source/write/delete.txt

```diff
@@ -0,0 +1,197 @@
+
+.. meta::
+   :keywords: remove, drop, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+php-library+} to remove
+documents from a MongoDB collection by performing **delete operations**.
+
+A delete operation removes one or more documents from a MongoDB collection.
+You can perform a delete operation by using the ``MongoDB\Collection::deleteOne()``
+or ``MongoDB\Collection::deleteMany()`` methods.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To access this collection
+from your PHP application, instantiate a ``MongoDB\Client`` that connects to an Atlas cluster
+and assign the following value to your ``$collection`` variable:
+
+.. literalinclude:: /includes/write/delete.php
+    :language: php
+    :dedent:
+    :start-after: start-db-coll
+    :end-before: end-db-coll
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Delete Operations
+-----------------
+
+You can perform delete operations by using the following methods:
+
+- ``MongoDB\Collection::deleteOne()``, which deletes *the first document*
+  that matches the search criteria
+- ``MongoDB\Collection::deleteMany()``, which deletes *all documents* that
+  match the search criteria
+
+Each delete method requires a **query filter** document, which specifies the
+search criteria to determine which documents to select for removal.
+For more information about query filters, see the 
+:manual:`Query Filter Documents section </core/document/#query-filter-documents>` in
+the {+mdb-server+} manual.
+
+Delete One Document
+~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``deleteOne()`` method to remove a document in
+the ``restaurants`` collection that has a ``name`` value of ``'Ready Penny Inn'``:
+
+.. literalinclude:: /includes/write/delete.php
+   :start-after: start-delete-one
+   :end-before: end-delete-one
+   :language: php
+   :dedent:
+
+Delete Multiple Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``deleteMany()`` method to remove all documents
+in the ``restaurants`` collection that have a ``borough`` value of ``'Brooklyn'``:
+
+.. literalinclude:: /includes/write/delete.php
+   :start-after: start-delete-many
+   :end-before: end-delete-many
+   :language: php
+   :dedent:
+
+Modify the Delete Operation
+---------------------------
+
+You can modify the behavior of the ``MongoDB\Collection::deleteOne()`` and
+``MongoDB\Collection::deleteMany()`` methods by by passing an array that
+specifies option values as a parameter. The following table describes the
+options you can set in the array:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Option
+     - Description
+
+   * - ``collation``
+     - | Specifies the kind of language collation to use when sorting
```

Comment:
Is "sorting" the correct term to use here? For a delete operation I'd expect `collation` to only impact matching.

Even for a `find` command, I expect `collation` would be used for matching _and_ sorting.

Reply:
Changed to "comparing strings," since I think that better describes what happens during matching and sorting